### ISSUE TITLE
False positives for mac addresses

### ISSUE THREAD

**Issue by jokay** [textlint/textlint#827]
**What version of textlint are you using?**

`12.0.2`

**What file type (Markdown, plain text, etc.) are you using?**

Markdown

**What did you do? Please include the actual source code causing the issue.**

```md
| ID | Mac               |
| -- | ------------------|
| 1  | 52:85:1a:31:2a:3d |
```

**What did you expect to happen?**

Shouldn't report anything.

**What actually happened? Please include the actual, raw output from textlint.**

```sh
35:53  ✓ error  Incorrect usage of the term: “3d”, use “3D” instead  terminology
```

I know I could use backticks for the mac address but I don't think this is correct *(beside having to update a lot of markdown documents)*.

Or is there a way to just ignore some terminologies like "3D" ?

**Comment by azu** [comment]
Probably, this issue is related to https://github.com/sapegin/textlint-rule-terminology.
Please create an issue to https://github.com/sapegin/textlint-rule-terminology.
